en/editor/extension/scene-script.md

```diff
@@ -0,0 +1,73 @@
+# Call the engine API and project script
+
+In the plugin you can declare a special script file (scene script), the script and the script in the project ( `assets` directory script) has the same environment, that is in this script can call the engine API and other projects Script to achieve:
+
+- Traverse the nodes in the scene to get or change the data
+- Call the other scripts in the project to complete the job
+
+## Register the scene script
+
+First add a `scene` field to the `contributions` property of `pacakge.json`, the value of which is the path to a script file, relative to the extension package directory.
+
+```json
+{
+    "name": "engine",
+    "contributions": {
+        "scene": {
+            "script": "./scene.js"
+        }
+    }
+}
+```
+
+## Write the scene script
+
+`scene.js` needs to be defined in this form.
```

Comment:
`scene.js` needs to be defined in this form.
-->
`scene.js` needs to contain specific code. Example: